docs/source/config_syntax.md

```diff
@@ -12,10 +12,10 @@ Content:
 
 - [A basic example](#a-basic-example)
 - [Syntax examples explained](#syntax-examples-explained)
-  - [`@` to interpolate with Python objects](#1--to-interpolate-with-python-objects)
```

Comment:
We had discussed the wording here earlier, on further thought perhaps this should be "@ to reference Python objects in configurations" and then using the word reference/referencing elsewhere to describe what @ is doing.

Reply:
sure, sounds good to me.

Reply:
Hi @ericspod @wyli ,

Updated in the latest commit.
Could you please help review it again?

Thanks in advance.